gaek/ndb_json.py

```diff
@@ -120,9 +127,25 @@ def encode_basevalue(obj):
 }
 
 
+ALTERNATIVE_NDB_TYPE_ENCODING = {
+  ndb.Key: encode_key_as_pair,
+}
+
+
 class NdbEncoder(json.JSONEncoder):
   """Extend the JSON encoder to add support for NDB Models."""
 
+  # Whether to fetch entities specified by ndb.Key or to return them as pairs.
+  fetch_models=True
+
+  def __init__(self, skipkeys=False, ensure_ascii=True, check_circular=True, allow_nan=True, sort_keys=False,
+               indent=None, separators=None, encoding='utf-8', default=None, fetch_models=True):
+
+    self.fetch_models = fetch_models
+
+    super(NdbEncoder, self).__init__(skipkeys, ensure_ascii, check_circular, allow_nan, sort_keys, indent, separators,
```

Comment:
`super` can get weird, so I prefer calling the parent class' `__init__` directly as in my example above.